assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/templates/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
```

Comment:
It is recommended to not set the namespace field in charts, unless you want to create the resource in the namespace different than the release namespace, so the default namespace from the kubeconfig context can be used. So please remove this field.